pkgs/by-name/st/steam-play-none/package.nix

```diff
@@ -0,0 +1,43 @@
+{ fetchFromGitHub
+, stdenvNoCC
+, lib
+, bash
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "Steam-Play-None";
+  version = "0-unstable-2022-12-15";
+  src = fetchFromGitHub {
+    repo = finalAttrs.name;
+    owner = "Scrumplex";
+    rev = "42e38706eb37fdaaedbe9951d59ef44148fcacbf";
+    hash = "sha256-sSHLrB5TlGMKpztTnbh5oIOhcrRd+ke2OUUbiQUqoh0=";
+  };
+  buildInputs = [ bash ];
+  strictDeps = true;
+  outputs = [ "out" "steamcompattool" ];
+  installPhase = ''
+    runHook preInstall
+
+    # Make it impossible to add to an environment. You should use the appropriate NixOS option.
+    # Also leave some breadcrumbs in the file.
+    echo "${finalAttrs.name} should not be installed into environments. Please use programs.steam.extraCompatPackages instead." > $out
+
+    install -Dt $steamcompattool compatibilitytool.vdf toolmanifest.vdf
+    install -Dt $steamcompattool -m755 launch.sh
+
+    runHook postInstall
+
+  '';
+
+  meta = {
+    description = ''
```

Comment:
Looks too long for `description`, probably put the instructions in `longDescription`?

Reply:
I don't agree. I took this directly from https://github.com/NixOS/nixpkgs/blob/29fe096e2030dd8741bb97dc1c06547816ac957f/pkgs/by-name/pr/proton-ge-bin/package.nix#L46-L50

Reply:
According to [meta.chapter.md](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description), `description` should be one-line.

I guess I'll leave it as is and if anyone wants to fix that, create a PR that targets both packages.

Reply:
hmm, then proton-ge-bin should be updated then